CHANGELOG.md

```diff
@@ -1,20 +1,23 @@
 # CDTDatastore CHANGELOG
 
+## Unreleased
+
+- [NEW] HTTP Interceptor API. See [Http Interceptors](https://github.com/cloudant/CDTDatastore/blob/master/doc/httpinterceptors.md) for details.
+
```

Comment:
Above the "new" feature, add `[IOS] Minimum iOS version now iOS 7. We now use NSURLSession.`